zerver/openapi/zulip.yaml

```diff
@@ -4532,18 +4532,19 @@ paths:
 
                                             [calc-full-member]: /api/roles-and-permissions#determining-if-a-user-is-a-full-member
                                     create_multiuse_invite_group:
-                                      type: integer
-                                      description: |
-                                        The ID of the [user group](/api/get-user-groups) whose members are
-                                        allowed to create [reusable invitation
-                                        links](/help/invite-new-users#create-a-reusable-invitation-link)
-                                        to the organization.
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            The ID of the [user group](/api/get-user-groups) whose members are
+                                            allowed to create [reusable invitation
+                                            links](/help/invite-new-users#create-a-reusable-invitation-link)
+                                            to the organization.
 
-                                        This setting can currently only be set to user groups that are
-                                        system groups, except for the system groups named
-                                        `"role:internet"` and `"role:owners"`.
+                                            This setting can currently only be set to user groups that are
+                                            system groups, except for the system groups named
+                                            `"role:internet"` and `"role:owners"`.
```

Comment:
We recently removed these lines from some of the existing settings. So, maybe these can be removed too, probably in a  separate commit.

Reply:
Yeah, that detail is now spelled out in the main API docs page on group-setting values. I think we need this text to be updated to the standard copy for such values, which also links to that main API docs page.

Reply:
Removed in the same commit for now, since it made sense to remove it when adding the [group-setting-value] reference.